src/LanguageServer/Impl/Implementation/Server.WorkspaceSymbols.cs

```diff
@@ -152,7 +158,11 @@ await GetModuleVariablesAsync(entry as ProjectEntry, opts, @params.query, 50, ca
                 res.children = Array.Empty<DocumentSymbol>();
             }
 
-            var loc = m.Locations.FirstOrDefault(l => !string.IsNullOrEmpty(l.FilePath));
+            var loc = m.Locations
+                .MaybeEnumerate()
+                .WhereNotNull()
```

Comment:
It is better to filter out `null` locations in the `MemberResult` itself, though I don't know when it actually can be null.

Reply:
OK